Update README for services (#864)
blockscout/blockscout-rs: smart-contract-verifier/README.md

# <h1 align="center"> Smart-contract Verifier </h1>

> **Smart-contract verifier** - service for verification of EVM based contracts. 
> Its basic idea is to accept bytecode to be verified and potential source files as input and return whether those files and bytecode correspond to each other.
> 
> Is a backbone service for everything related to smart-contract verification in blockscout.
> Is required if you want to have smart-contract verification functionality on your instance.

Reviewer: GPT proof-reading, wdyt?

```suggestion
**Smart-contract verifier** is a service for verifying EVM-based contracts. The primary function of this service is to receive bytecode and potential source files as inputs and determine whether the files and the bytecode correspond to each other.

This service serves as the core component for all activities related to smart-contract verification in BlockScout. It is essential for enabling smart-contract verification functionality on your instance.
```

Author: Applied, thanks!

## Requirements
No additional dependencies

## How to enable
Set the following ENVs on blockscout instance:
- `MICROSERVICE_SC_VERIFIER_ENABLED=true`
- `MICROSERVICE_SC_VERIFIER_URL={service_url}`
- `MICROSERVICE_SC_VERIFIER_TYPE=sc_verifier`

## Envs

> Here we describe only service specific variables. Variables common for all services can be found [here](../docs/common-envs.md).

Reviewer: GPT proof-reading
```suggestion
Here, we describe variables specific to this service. Variables common to all services can be found [here](../docs/common-envs.md).
```
